src/main/java/pl/zaprogramujzycie/mma/controllers/MedicineController.java

```diff
@@ -64,8 +68,8 @@ ResponseEntity<MedicineResponse> createMedicine(@AuthenticationPrincipal final P
             @ApiResponse(responseCode = "500", description = "Internal server error")
     })
     @GetMapping("/{id}")
-    ResponseEntity<MedicineResponse> findById(@AuthenticationPrincipal final Principal principal, @PathVariable final long id) {
-            return null;
+    ResponseEntity<MedicineResponse> findById(final Principal principal, @PathVariable final long id) {
+        return ResponseEntity.ok(service.findById(id, principal));
```

Comment:
Jak juz masz tego logger to mzoe jakis sensowny log na wejsciu do metody?

Reply:
Już nie mam loggera, bo jeszcze wczoraj przerzuciłam się na lombokowy po przejrzeniu brancha security.